mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -13,8 +13,11 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 	FIELD field_29847 RADIANS_PER_DEGREE F
 	FIELD field_29848 DEGREES_PER_RADIAN F
 	FIELD field_29849 EPSILON F
+	FIELD field_29850 FLOOR_DOWNSHIFT I
+	FIELD field_29851 FLOOR_UPSHIFT F
 	FIELD field_29853 HALF_PI_RADIANS_SINE_TABLE_INDEX J
 	FIELD field_29856 DEGREES_TO_SINE_TABLE_INDEX F
+	FIELD field_29859 SINE_TABLE_SIZE I
```

Comment:
It's the size for the ARC**SINE** and CO**SINE** tables. I picked the name because they both have "SINE" in the name, not because it belongs to the other SINE table.

Reply:
It's still confusing since you'd assume `SINE_TABLE_SIZE` is the size of `SINE_TABLE`.